DataModel/Protocol/Protocol-definition.yml

```diff
@@ -0,0 +1,285 @@
+Protocol:
+  description: Describes the protocols followed in a study, including details such
+    as the cell capture system, library protocol, the chemistry kit and sampling technique
+    used, as well as sequencing related information.
+  properties:
+    donor_id:
+      description: Foreign key to map to donor ID
+      type: string
+      format: alphanumeric
```

Comment:
@ishitasati11 Is there a reason as to why we need to specify `format`? In my opinion, `type` should suffice